include/roboteam_ai/control/positionControl/PathPointNode.h

```diff
@@ -0,0 +1,40 @@
+//
+// Created by ratoone on 20-02-20.
+//
+
+#ifndef RTT_PATHPOINTNODE_H
+#define RTT_PATHPOINTNODE_H
+
+#include <roboteam_utils/Vector2.h>
+
+namespace rtt::ai::control{
+/**
+ * A node in the path tree generation. Has a position and a parent
+ */
+class PathPointNode {
+private:
+    Vector2 position;
+    PathPointNode *parent = nullptr;
```

Comment:
which is default
```suggestion
    PathPointNode *parent;
```

Reply:
UB if I don't do this

Reply:
Nope, it's not UB because you still construct the class itself, meaning it gets default initialized to 0b000000...000

Reply:
Depends on what happens in the implementation. However I would consider init  with nullptr good practice